exonum-java-binding-qa-service/src/test/java/com/exonum/binding/qaservice/transactions/CreateCounterTxIntegrationTest.java

```diff
@@ -135,19 +120,22 @@ void executeNewCounter() throws CloseFailuresException {
   @RequiresNativeLibrary
   void executeAlreadyExistingCounter() throws CloseFailuresException {
     try (Database db = MemoryDb.newInstance();
-         Cleaner cleaner = new Cleaner()) {
+        Cleaner cleaner = new Cleaner()) {
       Fork view = db.createFork(cleaner);
+
       String name = "counter";
       Long value = 100500L;
-      HashCode nameHash = Hashing.defaultHashFunction()
-          .hashString(name, UTF_8);
+      HashCode nameHash = defaultHashFunction().hashString(name, UTF_8);
 
       // Add a counter with the given name and initial value to both indices manually.
       createCounter(view, name, value);
 
       // Execute the transaction, that has the same name.
       CreateCounterTx tx = new CreateCounterTx(name);
-      tx.execute(view);
+      TransactionContext context = TransactionContext.builder()
```

Comment:
Here and elsewhere: Shan't we create a tests-wide method `aContext(Fork) -> TransactionContext.Builder` or even `createContext(Fork) -> TransactionContext` that sets defaults?